Weather App/Common/TypeAliases.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  TypeAliases.swift
+//  Weather App
+//
+//  Created by Ivan Stajcer on 03.09.2021..
+//
+
+import Foundation
+
+typealias EmptyCallback = ()->Void
```

Comment:
Watch for spacing. Its `() -> Void `. Should be modified in the whole project, for all closures.

Reply:
I will fix the spacing.